src/auto-aspect-executor.ts

```diff
@@ -56,12 +56,19 @@ export class AutoAspectExecutor implements OnModuleInit {
     const metadataKey = this.reflector.get(ASPECT, lazyDecorator.constructor);
     // instance에 method names 를 순회하면서 lazyDecorator.wrap을 적용함
     for (const methodName of methodNames) {
```

Comment:
I think it will be easier to read if I change the name of the methodNames variable to propertyKeys

Reply:
That's right!
When I read the code, this has caused me some confusion. I will modify it together. 😄